src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,94 @@
+.container {
+  display: grid;
+  grid-template-columns: repeat(4, auto);
+  justify-content: center;
+  padding-top: 50px;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 32px 16px 16px;
+  margin-right: 48px;
+  margin-bottom: 46px;
+
+  border: 1px solid $card-border-color;
+  border-radius: 5px;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &__image {
+    align-self: center;
+    height: 134px;
+    width: 160px;
+
+    margin-bottom: 40px;
+  }
+
+  &__product-name {
+    margin-bottom: 4px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $basic-text-color;
+  }
+
+  &__product-code {
+    margin-bottom: 16px;
+    font-size: 10px;
+    line-height: 14px;
+    color: $price-and-code-color;
+  }
+
+  &__price {
+    margin-bottom: 16px;
+    line-height: 18px;
+    @extend %flex-align-center-space-between;
+  }
+
+  &__price-word {
+    font-size: 12px;
+    line-height: 18px;
+    color: $price-and-code-color;
+  }
+
+  &__price-value {
+    font-weight: 700;
+    color: $basic-text-color;
+  }
+
+  &__rating {
+    margin-bottom: 24px;
+    @extend %flex-align-center-space-between;
+  }
+
+  &__reviews {
+    font-size: 10px;
+    line-height: 14px;
+    color: $basic-text-color;
+  }
+
+  &__button-buy {
+    padding: 11px;
+    display: flex;
+    background-color: $blue-color;
+    border-radius: 5px;
+    box-sizing: border-box;
```

Comment:
remove it from everywhere and apply in * selector

Reply:
done
![image](https://user-images.githubusercontent.com/126285904/230630910-a16ec352-d326-4854-a3bb-6a1b1f5ff40d.png)
![image](https://user-images.githubusercontent.com/126285904/230631029-9d751b40-3f76-4ee0-bc8f-88ca15eb11ca.png)
![image](https://user-images.githubusercontent.com/126285904/230631298-c1391488-e058-4d28-9970-1a71b773fd75.png)

but it looks like crutches